extensions/cornerstone/src/init.js

```diff
@@ -187,11 +240,25 @@ export default function init({
                 label,
               });
 
-              MeasurementService.update(
-                updatedMeasurement.id,
-                updatedMeasurement,
-                true
-              );
+              nlApi
+                .patch(`/api/measurement/${updatedMeasurement.id}/`, { label })
+                .then(() => {
+                  UINotificationService.show({
+                    message: 'The measurement label has been updated',
+                    type: 'info',
+                  });
+                  MeasurementService.update(
+                    updatedMeasurement.id,
+                    updatedMeasurement,
+                    true
+                  );
```

Comment:
I think we don't need these UI notifications. I think we only need UI Notifications for the failure case with respect to measurements.